universe/Conditions.cpp

```diff
@@ -666,6 +674,12 @@ unsigned int Turn::GetCheckSum() const {
     return retval;
 }
 
+std::unique_ptr<Condition> Turn::Clone() const override {
+    auto retval = std::make_unique<Turn>(m_low ? std::move(m_low->Clone()) : nullptr,
```

Comment:
`std::move` probably unnecessary since it's already an rvalue

Reply:
Fixed.